Makefile

```diff
@@ -1,9 +1,10 @@
+# Y - in response to Would you like to add the OpenAPI addon?
```

Comment:
@sanG-github I think we don't need this options, just automatically add `OpenAPI` whenever the variant is API

Reply:
This `Makefile` is for test purposes only, it doesn't impact our template flow. @suho

Reply:
@sanG-github While it doesn't impact our template, it's unnecessary to have this change as we did not ask the question "Would you like to add the OpenAPI addon?"

Reply:
Ah okay I got the idea, updated it in 71c5dd2 ✅